plugins/a11y-keyboard/msd-a11y-keyboard-atspi.h

```diff
@@ -0,0 +1,55 @@
+/* -*- Mode: C; tab-width: 8; indent-tabs-mode: nil; c-basic-offset: 8 -*-
+ *
+ * Copyright (C) 2020 Colomban Wendling <cwendling@hypra.fr>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+
+#ifndef __MSD_A11Y_KEYBOARD_ATSPI_H
+#define __MSD_A11Y_KEYBOARD_ATSPI_H
+
+#include <glib-object.h>
+
+G_BEGIN_DECLS
+
+#define MSD_TYPE_A11Y_KEYBOARD_ATSPI            (msd_a11y_keyboard_atspi_get_type ())
+#define MSD_A11Y_KEYBOARD_ATSPI(o)              (G_TYPE_CHECK_INSTANCE_CAST ((o), MSD_TYPE_A11Y_KEYBOARD_ATSPI, MsdA11yKeyboardAtspi))
+#define MSD_A11Y_KEYBOARD_ATSPI_CLASS(k)        (G_TYPE_CHECK_CLASS_CAST((k), MSD_TYPE_A11Y_KEYBOARD_ATSPI, MsdA11yKeyboardAtspiClass))
+#define MSD_IS_A11Y_KEYBOARD_ATSPI(o)           (G_TYPE_CHECK_INSTANCE_TYPE ((o), MSD_TYPE_A11Y_KEYBOARD_ATSPI))
+#define MSD_IS_A11Y_KEYBOARD_ATSPI_CLASS(k)     (G_TYPE_CHECK_CLASS_TYPE ((k), MSD_TYPE_A11Y_KEYBOARD_ATSPI))
+#define MSD_A11Y_KEYBOARD_ATSPI_GET_CLASS(o)    (G_TYPE_INSTANCE_GET_CLASS ((o), MSD_TYPE_A11Y_KEYBOARD_ATSPI, MsdA11yKeyboardAtspiClass))
+
+typedef struct MsdA11yKeyboardAtspiPrivate MsdA11yKeyboardAtspiPrivate;
+
+typedef struct
+{
+        GObject                         parent;
+        MsdA11yKeyboardAtspiPrivate    *priv;
+} MsdA11yKeyboardAtspi;
```

Comment:
You can use boilerplate.

Reply:
Good idea, done.  I tend to forget these handy macro were added to GObject as I started using it way before that, so got used to writing the boilerplate by hand…